tests/vmcheck/multitest.py

```diff
@@ -115,6 +119,7 @@ def flush(self):
 
         rcs = "PASS" if rc == 0 else ("FAIL (rc %d)" % rc)
         print("%s: %s" % (rcs, self.test))
+        print("Execution took {} seconds".format(int(endtime - self._starttime)))
```

Comment:
Minor: I'd just compress these two `print` statements into a single one to make it clear which test took this number of seconds. E.g. something like `print("{}: {} (took {}s)".format(rcs, self.test, int(endtime - self._starttime))`?